src/snowflake/cli/plugins/snowpark/package/manager.py

```diff
@@ -64,7 +73,7 @@ def upload(file: Path, stage: str, overwrite: bool):
 
 
 def create(zip_name: str):
-    file_name = zip_name if zip_name.endswith(".zip") else f"{zip_name}.zip"
+    file_name = f"{get_package_name(zip_name)}.zip"
```

Comment:
Should we moved the `.zip`  suffix to `get_package_name` function?

Reply:
`get_package_name` is used to extract name, that we will use to resolve dependencies etc. - so for example, takes the name from github repo url, zipname etc. - it seems way too early to add `.zip` suffix there